### ISSUE TITLE
[Adaptive Cards iOS - Flight Itinerary]: Voice over is convey the incorrect information about the total amount under the flight itinerary screen.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#7577]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.3.1-beta.20220620.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS: 15.5
Application: Adaptive Cards
App Version: 1.0 (2.4.0-beta.20220612.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Activate Voice over
4. Navigate to "V1.0" control and select it.
5. Navigate to 'Scenarios' control and activate it.
6. Navigate to 'FlightItinerary.json' and activate it.
7. Navigate to the 'Total Amount' and observe the issue.

**Actual Result:**
Voice over does convey the incorrect information about the total amount under 'Flight Itinerary' screen. It is announcing as **'Dollar sign four comma thirty two period fifty-four.'**

**Expected Result:**
Voice over should convey the correct information about the total amount under 'Flight itinerary' screen and it should announce as **'Dollar sign Four thousand thirty two period fifty four.'**
**User Experience:**
It will impact the screen reader user if the voice over does convey the incorrect information for the amount as user will not able to understand about the amount.

**MAS Reference:**
https://liquid.microsoft.com/Web/Object/Read/MS.Accessibility/Requirements/01.03.01

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

### Screenshots

https://user-images.githubusercontent.com/95913748/176214623-a24ce6d9-3ed5-4857-a3be-2b3f41db8c56.MP4

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Closing this issue as of now this issue is tracking with below URL.

https://github.com/microsoft/AdaptiveCards-Mobile/issues/39
